runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -121,11 +131,16 @@ static void chpl_gpu_launch_kernel_help(int ln,
 
   CHPL_GPU_DEBUG("Creating kernel parameters\n");
 
+  int* gpu_alloc_map = chpl_malloc(nargs*sizeof(int));
```

Comment:
Huh, I guess we weren't cleaning up after ourselves before?

I'd probably rename it to something wordy but more self documenting like: `was_memory_dynamically_allocated_for_kernel_param`.